packages/fxa-event-broker/src/queueworker/queueworker.service.ts

```diff
@@ -54,22 +59,22 @@ export class QueueworkerService
     this.queueName = configService.get('serviceNotificationQueueUrl') as string;
     this.disableQueueWorker =
       this.configService.get<boolean>('disableQueueWorker') ?? false;
-    if (this.queueName.includes('localhost:4100')) {
-      AWS.config.update({
-        accessKeyId: 'fake',
-        ['endpoint' as any]: 'localhost:4100',
-        secretAccessKey: 'fake',
-        sslEnabled: false,
-      });
-    }
     this.topicPrefix = configService.get('topicPrefix') as string;
 
     const region = extractRegionFromUrl(this.queueName);
     this.region = region || 'us-east-1';
-    this.sqs = new SQS({ region: this.region });
+    this.sqs = new SQSClient({
+      ...(this.queueName.includes('localhost:4100') && {
+        accessKeyId: 'fake',
+        ['endpoint' as any]: 'localhost:4100',
+        secretAccessKey: 'fake',
+        sslEnabled: false,
+      }),
+      region: this.region,
+    });
```

Comment:
I can't find an exact solution in the migration docs -- it kind of seems like I need to pass this config into each AWS client.  Is this the only AWS client the event broker needs this config for?

Reply:
This is the correct way, it's documented by AWS here with a note on the change from 2.x
https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/global-config-object.html